assets/js/components/notifications/UnsatisfiedScopesAlert.js

```diff
@@ -76,13 +80,29 @@ export default function UnsatisfiedScopesAlert() {
 			new RegExp( '//oauth2|action=googlesitekit_connect', 'i' )
 		)
 	);
+	const tempPersistedPermissionsError = useSelect( ( select ) =>
+		select( CORE_FORMS ).getValue(
+			FORM_TEMPORARY_PERSIST_PERMISSION_ERROR,
+			'permissionsError'
+		)
+	);
 	const unsatisfiedScopes = useSelect( ( select ) =>
 		select( CORE_USER ).getUnsatisfiedScopes()
 	);
+
+	const connectURLData = {
+		redirectURL: global.location.href,
+	};
+	if ( tempPersistedPermissionsError?.data ) {
+		connectURLData.additionalScopes =
+			tempPersistedPermissionsError.data?.scopes;
+		connectURLData.redirectURL =
+			tempPersistedPermissionsError.data?.redirectURL ||
+			global.location.href;
+	}
```

Comment:
I think this would be easier to read as a ternary as it provides a clear object definition for each case:

```suggestion
	const connectURLData = tempPersistedPermissionsError?.data
		? {
				additionalScopes: tempPersistedPermissionsError.data?.scopes,
				redirectURL:
					tempPersistedPermissionsError.data?.redirectURL ||
					global.location.href,
		  }
		: {
				redirectURL: global.location.href,
		  };
```